### PR TITLE
add community-membership.md

### PR DESCRIPTION
**Pull request by gingerkidney**

### CODE REVIEW

**Comment by gingerkidney** [comment]
@dcalvin @winkyao PTAL

File: community-membership.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Community Membership
+
+## Maintainer
+
+Maintainers are the planners and designers of the project, with the authority to merge branches into the master. Maintainers come from Committers. They must demonstrate good judgment and responsibility for the health of the subproject. The maintainer must set up the technical direction directly or by delegating these responsibilities, and make or approve design decisions for the subproject.
+
+### How to become a Maintainer:
+
+- Demonstrate a deep understanding of TiDB's technical goals and directions
```
Comment (winkyao) [inline community-membership.md]: TiKV's

File: community-membership.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+# Community Membership
+
+## Maintainer
+
+Maintainers are the planners and designers of the project, with the authority to merge branches into the master. Maintainers come from Committers. They must demonstrate good judgment and responsibility for the health of the subproject. The maintainer must set up the technical direction directly or by delegating these responsibilities, and make or approve design decisions for the subproject.
```
Comment (dcalvin) [inline community-membership.md]:
```suggestion
Maintainers are the planners and designers of the project, with the authority to merge branches into the master. Maintainers come from Committers. They must demonstrate both good judgment and responsibility towards the health of the subproject. Maintainers must set up the technical direction and make or approve design decisions for the subproject - either directly or by delegating these responsibilities.
```

File: community-membership.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+## Committer
+
+Committers are recommended by Maintainers or the PMC as Contributors with outstanding contributions to TiKV. A committer must have completed at least one feature or fix a major bug independently.
+
+### How to become a Committer:
+
+- Demonstrated good technical judgment
+- Assumed the Reviewer role in at least two or more subprojects
+- Reviewed or merged a certain amount of TiKV code
```
Comment (dcalvin) [inline community-membership.md]: How much exactly... I see K8s has an exact number @winkyao @gingerkidney

File: community-membership.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+- Participate in Committer meetings on a regular basis and have knowledge priority regarding project development
+
+## Reviewer
+
+Reviewers are responsible for reviewing the code submitted by the community to ensure the quality and correctness; generated out of Contributors.
+
+### How to become a Reviewer:
+
+- Familiar with the codebase
+- Contributed a certain amount of code to TiKV subprojects
```
Comment (dcalvin) [inline community-membership.md]: how much?

File: community-membership.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
+# Community Membership
+
+## Maintainer
+
+Maintainers are the planners and designers of the project, with the authority to merge branches into the master. Maintainers come from Committers. They must demonstrate good judgment and responsibility for the health of the subproject. The maintainer must set up the technical direction directly or by delegating these responsibilities, and make or approve design decisions for the subproject.
+
+### How to become a Maintainer:
+
+- Demonstrate a deep understanding of TiKV's technical goals and directions
```
Comment (dcalvin) [inline community-membership.md]:
```suggestion
- Demonstrated a deep understanding of TiKV's technical goals and directions
```

File: community-membership.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+- Actively organize technical seminars and propose constructive projects
+- Can handle complex problems in the code implementation process
+- Get unanimous votes from Members of the PMC
```
Comment (dcalvin) [inline community-membership.md]:
```suggestion
- Obtained unanimous votes from Members of the PMC
```

File: community-membership.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
+# Community Membership
+
+## Maintainer
+
+Maintainers are the planners and designers of the project, with the authority to merge branches into the master. Maintainers come from Committers. They must demonstrate good judgment and responsibility for the health of the subproject. The maintainer must set up the technical direction directly or by delegating these responsibilities, and make or approve design decisions for the subproject.
+
+### How to become a Maintainer:
+
+- Demonstrate a deep understanding of TiKV's technical goals and directions
+- Actively organize technical seminars and propose constructive projects
```
Comment (dcalvin) [inline community-membership.md]: This sounds more like a responsibility...

File: community-membership.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+
+- Familiar with the codebase
+- Contributed a certain amount of code to TiKV subprojects
+- Performed excellently in the Special Interest Group (SIG)
+- Sponsored by at least two incumbent Committers
+
+### Privileges and responsibilities:
+
+- Ensure the correctness of the subproject code
+
```
Comment (dcalvin) [inline community-membership.md]: Why blank line in-between?

File: community-membership.md [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+- Received recognition from two incumbent Reviewers
+
+### Privileges and responsibilities:
+
+- Automatically qualify for entry to the Special Interest Group (SIG)
+
+## Contributor
+
+- Merged at least 1 PR in TiKV Repo
```
Comment (dcalvin) [inline community-membership.md]:
```suggestion
- Merged at least 1 PR in any Repo under [TiKV](https://github.com/tikv) 
```

File: community-membership.md (hunk 3 above)
Comment (gingerkidney) [reply community-membership.md]: It depends on different module

File: community-membership.md (hunk 4 above)
Comment (gingerkidney) [reply community-membership.md]: ditto

File: community-membership.md (hunk 7 above)
Comment (gingerkidney) [reply community-membership.md]: The Maintainer need to show their ability first, by deeply involved in TiKV and propose constructive projects

File: community-membership.md (hunk 8 above)
Comment (gingerkidney) [reply community-membership.md]: fixed

File: community-membership.md (hunk 7 above)
Comment (dcalvin) [reply community-membership.md]: Then maybe  "Actively involved in organizing technical seminars and proposing constructive subprojects"?

File: community-membership.md [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+
+- Develop technical design and decisions for subprojects
+- Set the technical directions and priorities for subprojects
+- Participate in product releases and roadmap development of the project
+- Continuously contribute to the project by mentoring Contributors, Reviewers, and Committers
+- Ensure the overall quality of the subprojects
+
+## Committer
+
+Committers are recommended by Maintainers or the PMC as Contributors with outstanding contributions to TiKV. A committer must have completed at least one feature or fix a major bug independently.
```
Comment (dcalvin) [inline community-membership.md]:
```suggestion
Committers are recommended by Maintainers or the PMC as Contributors with outstanding contributions to TiKV. A Committer must have completed at least one feature or fix a major bug independently.
```

File: community-membership.md [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+
+## Reviewer
+
+Reviewers are responsible for reviewing the code submitted by the community to ensure the quality and correctness; generated out of Contributors.
+
+### How to become a Reviewer:
+
+- Familiar with the codebase
+- Contributed a certain amount of code to TiKV subprojects
+- Performed excellently in the Special Interest Group (SIG)
```
Comment (dcalvin) [inline community-membership.md]:
```suggestion
- Performed excellently in a Special Interest Group (SIG)
```

File: community-membership.md [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+## Maintainer
+
+Maintainers are the planners and designers of the project, with the authority to merge branches into the master. Maintainers come from Committers. They must demonstrate both good judgment and responsibility towards the health of the subproject. Maintainers must set up the technical direction and make or approve design decisions for the subproject - either directly or by delegating these responsibilities.
+
+### How to become a Maintainer:
+
+- Demonstrated a deep understanding of TiKV's technical goals and directions
+- Actively organize technical seminars and propose constructive projects
+- Can handle complex problems in the code implementation process
+- Obtained unanimous votes from Members of the PMC
```
Comment (dcalvin) [inline community-membership.md]:
```suggestion
- Obtained unanimous votes from the PMC
```

File: community-membership.md (hunk 7 above)
Comment (gingerkidney) [reply community-membership.md]: done

**Pull request merged**